test/ctest_helpers.cmake

```diff
@@ -77,6 +77,24 @@ function(create_test_executable target)
         target_link_libraries(${target} ${CREATE_TEST_LIBS})
     endif()
 
+    if(WT_LINUX OR WT_DARWIN)
```

Comment:
Should do it for Windows too?

Reply:
Good point, after doing some research I found out that Windows does things completely differently and does not have a concept of rpath. Windows shared libraries are always looked up in directories that are in the current PATH, or by adding libraries to the PATH before running the program. 

That said, if need be we can extend this to Windows as well. I will not worry about it for now as it requires more investigation and is out of the scope of this ticket.